### PR TITLE
feat: simplified menu

### PR DESCRIPTION
**Pull request by gagdiez**
Currently the site uses multiple nested menus, as well as a sidebar combined with a top bar.

This commit removes the nested menus as well as the top bar, leaving only a one-level side menu, with the least amount of features posible to not confuse developers

In further iterations we should look into how to use the "Search Bar" (if necessary), as well as to make a decision on deprecate notifications, as they are part of the Social Media we no longer support.

Important: This PR requires some new components that were contributed to the near-discovery-components repo 

PR: https://github.com/near/near-discovery-components/pull/829

### CODE REVIEW

File: src/components/sidebar-navigation/Sidebar.tsx [hunk 1]
```diff
@@ -68,65 +96,131 @@ export const Sidebar = () => {
...
               </S.NavigationItem>
             </Tooltip>
 
-            <Tooltip content="Careers" side="right" disabled={tooltipsDisabled}>
-              <S.NavigationItem $active={false} $type="simple" href="https://careers.near.org/" target="_blank">
-                <i className="ph-bold ph-briefcase" />
-                <span>Careers</span>
-                <i className="ph-bold ph-arrow-square-out ms-auto" />
+            <Tooltip content="Blog" side="right" disabled={tooltipsDisabled}>
+              <S.NavigationItem $active={false} $type="featured" href="https://near.org/blog">
```
Comment (charleslavon) [inline src/components/sidebar-navigation/Sidebar.tsx:150]: `<S.NavigationItem $active={false} $type="featured" target="_blank" href="https://near.org/blog">`

Comment (charleslavon) [reply src/components/sidebar-navigation/Sidebar.tsx:150]: needs a target=_blank

**Review by charleslavon — CHANGES_REQUESTED** [review]
Perhaps the entire [DiscoverDrawer](https://github.com/near/near-discovery/blob/1873273bb1174da4db02dbbe4e2b0e26e404e383/src/components/sidebar-navigation/DiscoverDrawer.tsx#L32) should be removed in this PR as well, to avoid the jank that occurs when you click on `Applications`

**Comment by charleslavon** [comment]
@gagdiez do you intend to get rid of the nested `More` menu in this PR? I can trigger it on the preview link by clicking `News`

**Comment by gagdiez** [comment]
thanks for checking! indeed, there should be no second menu, will now fix what you found

**Review by charleslavon — APPROVED** [review]
:shipit:

**Pull request merged**